submissions/chri911/popup-menu/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <input type="checkbox" id="isexpanded">
+    <input type="checkbox" id="expanded">
+    <div class="container">
+        <div class="mail"><a href="#">Mail</a></div>
+        <div class="images"><a href="#">Images</a></div>
+        <div class="popup1">
+        <label for="isexpanded" id="popup"></label>
+            <div class="popup__menu">
+                <div class="popup__menu-wrapper">
+                <div class="flex">
+                <a href="#" class="item calendar">
+                    <img src="images/calendar.png" alt="">
```

Comment:
Avoid leaving empty `alt` attribute. [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) in the attribute section